### PR TITLE
frontend: Add webdriver troubleshooting to README

### PR DESCRIPTION
**Pull request by rebeccaalpert**
Added how to update webdriver to new version of Chrome.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -204,6 +204,13 @@ yarn run test-gui-openshift
 This will include the normal k8s CRUD tests and CRUD tests for OpenShift
 resources.
 
+If `yarn run test-gui` fails because your browser is too new, you can run:
```
Comment (spadgett) [inline README.md]: I would rephrase this to say:

> If you your local Chrome version doesn't match the Chromedriver version from the console dependencies, override the version with:

Comment (rebeccaalpert) [reply README.md]: Amended the commit with this.

**Comment by spadgett** [comment]
/override ci/prow/e2e-gcp-console

Docs only change
